7a_temp_coop_munge/src/data_parsers/parse_missouri_files.R

```diff
@@ -0,0 +1,85 @@
+
+# Parse the ~300 files within the `Bull_Shoals_Lake_DO_and_Temp.zip` file
+
+parse_Bull_Shoals_Lake_DO_and_Temp <- function(inind, outind) {
+
+  infile <- sc_retrieve(inind, remake_file = '6_temp_coop_fetch_tasks.yml')
```

Comment:
Likely a good reason for this, but wondering why the `remake_file` for this isn't `getters.yml` for this like it would be for other retrieve/gd_get targets.

Reply:
I am not sure if there is a good reason, other than to say that I followed the pattern present in all of the other data parsers. None of those use `getters.yml`.